templates/experimental-kube-vip/kustomization.yaml

```diff
@@ -2,115 +2,22 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
 resources:
-- ../cluster-template.yaml
+  - ../cluster-template.yaml
 patches:
-- patch: |-
```

Comment:
was this spacing change necessary? It makes this harder to review. Perhaps if it was done in separate commits it would be easier on the eyes.

Reviewed in ignore-whitespace mode.. https://github.com/kubernetes-sigs/cluster-api-provider-packet/pull/660/files?diff=unified&w=1

Reply:
This is my vscode's fault. Sorry. Figured I'd get it committed now so it stops doing it in the future.